src/crudeditor-lib/middleware/notifications.js

```diff
@@ -116,10 +117,10 @@ const eventsMiddleware = /* istanbul ignore next */ ({ i18n, modelDefinition })
         type: 'error',
         timeOut: ERROR_NOTIFICATION_TIMEOUT,
         message: (Array.isArray(action.payload) ? action.payload : [action.payload]).
-          filter(err => err && typeof err === 'object' && err.message).
-          map(({ message }) => message).
+          filter(err => err && typeof err === 'object').
+          map(({ id, message }) => getModelMessage(i18n, `model.error.${id}`, message)).
```

Comment:
**error** might have optional **payload** necessary for getting correct translation, see [documentation](https://github.com/OpusCapita/react-crudeditor/blob/issue-217/README.md#parsing-error-and-fieldinstance-validation-error) for details.